### ISSUE TITLE
Behavior change for lca search - 'search' on LCA databases does not use abundance Error

### ISSUE THREAD

**Issue by luizirber** [dib-lab/sourmash#803]
I ran https://github.com/luizirber/sourmash_resources again with new master, but got an error during `sourmash search` with a LCA database:

Error:
```
== This is sourmash version 2.3.2.dev6+g13dc11f.d20191218. ==
== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==

selecting specified query k=51
loaded query: CAVL1ANXX170419:7:1101:10005:9... (k=51, DNA)
loaded 1 databases.

Traceback (most recent call last):
  File "bin/sourmash", line 11, in <module>
    load_entry_point('sourmash', 'console_scripts', 'sourmash')()
  File "sourmash/sourmash/__main__.py", line 83, in main    cmd(sys.argv[2:])
  File "sourmash/sourmash/commands.py", line 558, in search
    args.best_only, args.ignore_abundance)
  File "sourmash/sourmash/search.py", line 37, in search_databases
    best_only=best_only)
  File "sourmash/lca/lca_utils.py", line 279, in search
    raise TypeError("'search' on LCA databases does not use abundance")
TypeError: 'search' on LCA databases does not use abundance
```

This worked before (in `2.x`), and is a change that landed with #556. Maybe warn that abundance is not used, instead of throwing an error?

To reproduce locally:
```
wget -c https://ibdmdb.org/tunnel/static/HMP2/WGS/1750/HSMA33OT.fastq.gz
wget -c -O genbank-k51.lca.json.gz https://osf.io/nemkw/download
sourmash compute -k 21,31,51 --scaled 2000 --track-abundance --name-from-first -o HSMA33OT.fastq.gz.sig HSMA33OT.fastq.gz
sourmash search -o out.csv --scaled 2000 -k 51 HSMA33OT.fastq.gz.sig genbank-k51.lca.json.gz 
```

I'll work on a smaller test case.

**Comment by luizirber** [comment]
cc @ctb

**Comment by luizirber** [comment]
#804 has a test triggering this

**Comment by ctb** [comment]
This was my mistake. I'm fixing it in #804.

**Comment by luizirber** [comment]
> This was my mistake. I'm fixing it in #804.

If it passed code review and is in master, it is everyone's mistake =]

**Comment by ctb** [comment]
On Thu, Dec 19, 2019 at 02:44:50PM -0800, Luiz Irber wrote:
> > This was my mistake. I'm fixing it in #804.
> 
> If it passed code review and is in master, it is everyone's mistake =]

I think the test you added is the right one :)

**Comment by luizirber** [comment]
Fixed by #804
